lib/Cmt.ml

```diff
@@ -134,7 +134,7 @@ let fmt cmt src ~wrap:wrap_comments ~fmt_code =
           fmt_asterisk_prefixed_lines asterisk_prefixed_lines
     else
       match split_asterisk_prefixed cmt with
-      | [""] -> str "(* *)"
```

Comment:
Should we keep an assert here ?
I don't like that but the `asterisk_prefixed_lines` can't handle `[""]`, it would format to `(**)`, which would parse as a documentation comment.

Reply:
The `[]` cannot work too.